option optimize_for = LITE_RUNTIME;

message FriendAddByNameRequest
{
 required string name = 1;
}
